hoomd/md/PotentialBondGPU.cuh

```diff
@@ -242,9 +242,10 @@ __global__ void gpu_compute_bond_forces_kernel(Scalar4* d_force,
     This is just a driver function for gpu_compute_bond_forces_kernel(), see it for details.
 */
 template<class evaluator>
-hipError_t gpu_compute_bond_forces(const kernel::bond_args_t& bond_args,
-                                   const typename evaluator::param_type* d_params,
-                                   unsigned int* d_flags)
+__attribute__((visibility("default"))) hipError_t
```

Comment:
Why does the visibility attribute need to be added here?

Reply:
I added it to be consistent with the others (pair, external, etc...). It would be needed for any C++ unit tests that use PotentialBond*GPU classes. With pybind11, the visibility is hidden so the methods are not available to executables linked to the `_md` library.